content/modules/redisgears/jvm/classes/readers/commandreader.md

```diff
@@ -0,0 +1,44 @@
+---
+Title: CommandReader
+linkTitle: CommandReader
+description: Run RedisGears functions on command.
+weight: 60
+alwaysopen: false
+categories: ["Modules"]
+---
+
+The `CommandReader` allows you to run RedisGears functions on command when you:
```

Comment:
There is also the CommandOverrider, I believe you do not have the most up to date library:
https://github.com/RedisGears/JVMPlugin/blob/master/gears_runtime/src/main/java/gears/readers/CommandOverrider.java